lib/ansible/modules/crypto/openssh_keypair.py

```diff
@@ -202,6 +205,12 @@ def _check_state():
 
         if _check_state():
             proc = module.run_command([module.get_bin_path('ssh-keygen', True), '-lf', self.path])
```

Comment:
```suggestion
            proc = module.run_command([module.get_bin_path('ssh-keygen', True), '-lf', self.path], check_rc=False)
```
Otherwise Ansible will fail the module when a return code != 0 is returned.

Reply:
(No idea where the modified space comes from.)

Reply:
But wouldn't that indicate an error?

Reply:
Yes, but the code below that is already checking for a non-zero return code.